pandas/tests/indexes/datetimes/test_datetime.py

```diff
@@ -368,3 +368,14 @@ def test_factorize_dst(self):
     def test_unique(self, arr, expected):
         result = arr.unique()
         tm.assert_index_equal(result, expected)
+
```

Comment:
can you move near the other tests?

also can you add all of the examples from the original issue (e.g. the working ones). so this is a complete example.

can you add a comment on what is being tested here.

Reply:
> can you move near the other tests?

Sorry, didn't quite get this one, but tried to move this closer to other tests that seem a little similar.

> also can you add all of the examples from the original issue (e.g. the working ones). so this is a complete example.

There wasn't a lot of stuff in the original issue, but I added what the person who posted that issue was attempting to do. 

> can you add a comment on what is being tested here.

Added a short comment.